introduction.md

```diff
@@ -0,0 +1 @@
+Dojo, I am Jacob, and I am trying to learn the best that I can. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.